### ISSUE TITLE
Teamcity

### ISSUE THREAD

**Issue by patolo** [marcells/node-build-monitor#9]
Is there a way of displaying the names of the branches using the service TeamCity.js?
I'm using git flow and would like to display feature branches.

**Comment by marcells** [comment]
I've no experiences with git flow. Do you know how to query the needed information with the [TeamCity REST API](https://confluence.jetbrains.com/display/TCD9/REST+API)?

**Comment by patolo** [comment]
Hi @marcells, I can't really find a nice way using the TeamCity API for this purpose. Hoping you have a good idea ;)

**Comment by marcells** [comment]
;-) I neither have (a lot of) experience with TeamCity, nor with gitflow. I don't find any time this weekend, but I will take a look at it next week.

**Comment by patolo** [comment]
I don't think we could use TeamCity like this without integration against CVS, Github.
